fuel/downloaders/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""Download modules for built-in datasets.
+
+Download methods accept a single argument, `save_directory`, which tells
+where to save the downloaded files.
+
+"""
+from fuel.downloaders.binarized_mnist import binarized_mnist
+
+__all__ = ('binarized_mnist',)
```

Comment:
Strange, I traced it back to https://github.com/GreenSteam/pep257/issues/62. Sounds a bit paranoid to me, but sure, let's keep the tuples then.